query-compiler/query-compiler/src/translate.rs

```diff
@@ -298,6 +299,9 @@ impl<'a, 'b> NodeTranslator<'a, 'b> {
                         RowSink::SingleRow(field) => {
                             *field.node_input_field(&mut node) = SelectionResult::new(fields);
                         }
+                        RowSink::SingleRowFilter(field) => {
+                            *field.node_input_field(&mut node) = SelectionResult::new(fields).filter();
+                        }
```

Comment:
Note: this is a fix for a bug in the query-compiler that wouldn't have allowed tests to run with SQL Server otherwise.